presto-main/src/test/java/io/prestosql/server/TestDynamicFilterService.java

```diff
@@ -247,6 +285,18 @@ public void testDynamicFilter()
         assertEquals(dynamicFiltersStageSupplier.getRequestCount(), 6);
         assertTrue(blockedFuture.isDone());
 
+        stats = dynamicFilterService.getDynamicFilteringStats(queryId, session);
+        assertEquals(stats.getNumDynamicFiltersCompleted(), 3);
+        assertEquals(stats.getNumRepartitionedDynamicFilters(), 3);
```

Comment:
please also assert `getNumRepartitionedDynamicFilters` and `getNumReplicatedDynamicFilters` after first `getDynamicFilteringStats` call